src/source/Ice/TurnConnectionStateMachine.c

```diff
@@ -60,30 +60,6 @@ PCHAR turnConnectionGetStateStr(UINT64 state)
     return TURN_STATE_UNKNOWN_STR;
 }
 
-STATUS checkTurnConnectionStateMachine(PTurnConnection pTurnConnection)
-{
-    ENTERS();
-    STATUS retStatus = STATUS_SUCCESS;
-    BOOL transitionReady = FALSE;
-
-    CHK(pTurnConnection != NULL && pTurnConnection->pStateMachine != NULL, STATUS_NULL_ARG);
-
-    // if a state transition is ready, tell the timer to kick the timer
-    CHK_STATUS(checkForStateTransition(pTurnConnection->pStateMachine, &transitionReady));
-
-    if (transitionReady) {
-        // dangerous to have any mutexes locked by timerqueue when entering this function
-        CHK_STATUS(timerQueueKick(pTurnConnection->timerQueueHandle, pTurnConnection->timerCallbackId));
```

Comment:
Is this Kick added to some other place?

Reply:
No it's been removed from all the places where it was.

Reply:
if this is the change caused the Axis P3354 not work properly due to the contention-  let  us add the details on the locks/threads?